### ISSUE TITLE
[REQUEST] Everybody's Golf/Hot Shots Golf: World Invitational

### ISSUE THREAD

**Issue by Poptheweasel100** [Electry/VitaGrafixPatchlist#50]
Surprised this hasn't been made, considering Hot Shots Golf is popular... Right? Unless patches have been made, and I'm blind, or they haven't been made because of glitches. Either way, if this is possible, could we get a native resolution patch? I love this game, and it being a blurry mess on my screen is something I want to fix.

**Comment by TheMostEve** [comment]
Up

**Comment by EnekoT2001** [comment]
Agreed, I made a request on the VitaGrafix repo a long time ago (before the patchlist repo was a thing). Basically the game breaks when you try to change the res (link [here](https://github.com/Electry/VitaGrafix/issues/14)).
Maybe is worth giving it a try with the new features of VG tho. I don't get why they used that resolution even games like uncharted uses a higher one...
